### PR TITLE
feat(icons): add icons for collections

### PR DESCRIPTION
**Pull request by nickgriffin1**
image

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @nickgriffin1 has signed the CLA. Thanks for the pull request!

File: src/icons/collections/__tests__/IconCollectionsAdd-test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import IconCollectionsAdd from '../IconCollectionsAdd';
+
+describe('icons/collections/IconCollectionsAdd', () => {
```
Comment (nickgriffin1) [inline src/icons/collections/__tests__/IconCollectionsAdd-test.js]: I'm not sure these unit tests are totally necessary. I can be convinced to remove them.

**Comment by booc0mtaco** [comment]
@nickgriffin1 can we get imports where the view boxes are "0 0 32 32"

File: src/icons/collections/__tests__/IconCollectionsAdd-test.js (hunk 1 above)
Comment (FuzzyTree) [reply src/icons/collections/__tests__/IconCollectionsAdd-test.js]: It looks like other Icons in this project use snapshots for testing icons. I think either way is ok.

Comment (nickgriffin1) [reply src/icons/collections/__tests__/IconCollectionsAdd-test.js]: I'll add snapshots. That's a good idea.

**Comment by nickgriffin1** [comment]
@booc0mtaco I plan on getting and updated the icon with a corrected one at a later date.

**Pull request merged**